pkg/controller/cleanup/cleanup.go

```diff
@@ -93,6 +96,7 @@ func (c *Controller) HandleCleanup() http.Handler {
 				OK:     false,
 				Errors: []error{fmt.Errorf("too early")},
 			})
+			return
```

Comment:
This is an existing nasty bug in the cleanup server.

Reply:
good catch